README.md

```diff
@@ -13,22 +13,40 @@ Our implementation is based on [CSBDEEP](http://csbdeep.bioimagecomputing.com) (
 
 ## Installation
 This implementation requires [Tensorflow](https://www.tensorflow.org/install/).
-We have tested Noise2Void using Python 3.7 and tensorflow-gpu 2.4.1.
+We have tested Noise2Void using Python 3.9 and TensorFlow 2.7 and 2.10.
 
-Note: If you want to use TensorFlow 1.15 you have to install N2V v0.2.1. N2V v0.3.0 supports TensorFlow 2 only.
+Note: If you want to use TensorFlow 1.15 you have to install N2V v0.2.1. N2V v0.3.* supports TensorFlow 2 only.
 
 #### If you start from scratch...
 We recommend using [miniconda](https://docs.conda.io/en/latest/miniconda.html).
 If you do not yet have a strong opinion, just use it too!
 
-After installing Miniconda, the following lines might are likely the easiest way to get Tensorflow and CuDNN installed on your machine (_Note:_ Macs are not supported, and if you sit on a Windows machine all this might also require some modifications.):
+After installing Miniconda, the following lines might are likely the easiest way to get Tensorflow and 
+CuDNN installed on your machine.
+_Note:_ Macs are not GPU-supported with TensorFlow.
 
+The [Tensorflow guidelines](https://www.tensorflow.org/install/pip) provide a step by step guide on how to install
+TensorFlow on the various operating system. 
+
+**For linux:**
 ```
-conda create -n n2v python=3.8 cudatoolkit=11.0 cudnn=8.0 -c conda-forge
+conda install -n n2v -c conda-forge cudatoolkit=11.2 cudnn=8.1.0
 conda activate n2v
-pip install tensorflow==2.4
+
+# export path to be able to use GPU
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$CONDA_PREFIX/lib/
```

Comment:
Should we add this convenient function:
```
mkdir -p $CONDA_PREFIX/etc/conda/activate.d
echo 'export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$CONDA_PREFIX/lib/' > $CONDA_PREFIX/etc/conda/activate.d/env_vars.sh
```
Taken from the tensorflow install instructions. 
Con: It hides away that the LD_LIBRARY_PATH is set.
Pro: It automatically sets the LD_LIBRARY_PATH up on env activation.

Reply:
Much much better than having the `export` everywhere...